services/movie_session.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+from django.db.models.functions import TruncDate
+
+import init_django_orm  # noqa: F401
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movie = Movie.objects.get(id=movie_id)
+
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall,
+        movie=movie
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet | MovieSession:
+    queryset = MovieSession.objects.all()
+
+    if session_date:
+        date_object = datetime.strptime(session_date, "%Y-%m-%d").date()
```

Comment:
Consider adding error handling for `datetime.strptime` to manage cases where `session_date` is not in the expected format. This will prevent the function from raising an exception if the input is invalid.